### PR TITLE
chore(testing): Move all testing deps into utils/rx-page-objects

### PR DESCRIPTION
**Pull request by Droogans**
JIRA:

### LGTMs
- [x] Dev LGTM

### CODE REVIEW

File: utils/rx-page-objects/.gitignore [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+!doc/layout.tmpl
+!doc/README.md
+node_modules/
+typings/
+
+# these are typescript now
+# eventually, all src/*.page.js files
+# can be ignored with a glob.
+src/rxComponent.js
+src/rxDatePicker.page.js
```
Comment (Droogans) [inline utils/rx-page-objects/.gitignore]: Oops, that's a problem. I'll fix this.

**Comment by parlarjb** [comment]
LGTM

**Comment by Droogans** [comment]
This PR needs https://github.com/rackerlabs/snappit-mocha-protractor/pull/53 to get merged and shipped to npm to fix these errors.

**Comment by Droogans** [comment]
Moving the relative location of the test files changes the relative path to the screenshots, which gets used in the name.

I think this is OK since I'll just need to delete the screenshots located in `utils/visual-regression` since they're now located in `visual-regression`.

**Pull request merged**